pylint/checkers/stdlib.py

```diff
@@ -599,6 +620,35 @@ def _check_lru_cache_decorators(self, decorators: nodes.Decorators) -> None:
                 confidence=interfaces.INFERENCE,
             )
 
+    def _check_dispatch_decorators(self, node: nodes.FunctionDef) -> None:
+        decorators_names_set: set[str] = set()
+        decorators_name_node_map: dict[str, nodes.NodeNG] = {}
+
+        decorators = node.decorators.nodes if node.decorators else []
+        for decorator in decorators:
```

Comment:
```suggestion
        for decorator in node.decorators:
```

We check for this in the calling code.